flyway/sql/V28__no_anonymous_on_registration_funnel_1.sql

```diff
@@ -0,0 +1,3 @@
+-- remove support for anonymous insert on registration_funnel_1
+REVOKE ALL ON mappa.registration_funnel_1 FROM anonymous;
+GRANT INSERT ON mappa.registration_funnel_1 to member;
```

Comment:
still don't quite understand this, but i'm happy to leave something that works. i imagine we'll need to take a more comprehensive review of the permissions and make sure everything is lining up in the future.